examples/firebase_example/.bazelrc

```diff
@@ -20,3 +20,16 @@ build --cxxopt='-std=gnu++14'
 # Firebase SPM support requires `-ObjC` linker option.
 # https://github.com/firebase/firebase-ios-sdk/blob/master/SwiftPackageManager.md#requirements
 build --linkopt='-ObjC'
+
+# Use the sandbox for rules_xcodeproj-specific builds.  rules_xcodeproj removes
+# the sandboxed spawn strategy (--spawn_strategy=remote,worker,local) by
+# default to speed up builds. However, to support setting C/C++ language
+# standard flags per target
+# (https://github.com/cgrindel/rules_swift_package_manager/issues/1079), we
+# need to generate multiple {cc,objc}_library targets based upon the type of
+# source files (e.g. C, C++, ObjC, ObjC++). When builds occur outside of the
+# sandbox, we can experience "error: duplicate interface definition for class
+# 'XXX'" errors. Until we can address the underlying error, clients that use
+# rules_swift_package_manager with rules_xcodeproj may need to use sandbox
+# builds.
```

Comment:
Is this fixable somehow either here or in the other rules? It feels like it would be a blocker for a lot of people since most places I've seen run unsandboxed builds for performance reasons.

Reply:
I tried several avenues to make this work without enabling sandboxing (e.g. use the `rules_swift_package_manager` modulemap instead of the `rules_swift` modulemap, disabling implicit modulemaps). @brentleyjones and I spoke yesterday about a change that might address this issue. In short, it turns out that clang only looks two directories down when searching for modulemap files. `rules_swift` generates their modulemaps at `path/to/Foo/_/module.modulemap` so that they won't be found when implicit module maps are enabled (default). We may want to update `generate_modulemap` in this repo to do the same. 

I would like to land this functionality as it does work properly. We can tackle the optimization of supporting no-sandbox builds as a separate PR.